client/js/lounge.js

```diff
@@ -427,7 +427,7 @@ $(function() {
 	var userStyles = $("#user-specified-css");
 	var settings = $("#settings");
 	var options = $.extend({
-		badge: false,
+		desktopNotifications: false,
```

Comment:
I know I'm getting close to the bikeshed here, but I think `browserNotifications` is closer to what it is than `desktopNotifications`. Don't take my word for it, I'll be fine with whatever opinion @YaManicKill has :-)